### PR TITLE
enable statistics

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
enable statistics

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
@BusyJay @siddontang @disksing @ngaut PTAL

File: src/main.rs [hunk 1]
```diff
@@ -181,6 +181,8 @@ mod tests {
         opts.set_filter_deletes(false);
         opts.set_report_bg_io_stats(true);
         opts.set_wal_recovery_mode(DBRecoveryMode::PointInTime);
+        opts.enable_statistics();
```
Comment (BusyJay) [inline src/main.rs]: This is not a test case.

File: src/rocksdb_options.rs [hunk 2]
```diff
@@ -505,6 +505,19 @@ impl Options {
...
         }
     }
+
+    pub fn enable_statistics(&mut self) {
+        unsafe {
+            rocksdb_ffi::rocksdb_options_enable_statistics(self.inner);
+        }
+    }
+
+    pub fn set_stats_dump_period_sec(&mut self, period: c_uint) {
```
Comment (BusyJay) [inline src/rocksdb_options.rs]: Use usize instead.

**Comment by siddontang** [comment]
How to disable ?

**Comment by zhangjinpeng1987** [comment]
@siddontang Enable when open db, and cannot disable.

**Comment by siddontang** [comment]
Got it.
LGTM

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**